src/language-js/parser-babel.js

```diff
@@ -65,6 +65,9 @@ function createParse(parseMethod, ...pluginCombinations) {
     // Inline the require to avoid loading all the JS if we don't use it
     const babel = require("@babel/parser");
 
+    const sourceType =
+      opts && opts.__babelSourceType === "script" ? "script" : "module";
```

Comment:
No need for `&&`. `opts` is always defined here.

Reply:
No... I forgot how. Maybe called by another parser

Reply:
Okay, if you say so.